Agenta-AI/agenta
# Test: delete created app variant

**Description:**
After the AB testing was finished, wrote a test to delete the created app variant.

**Test added:**
- Delete created app variant
- Delete created evaluation for ab-test
- Check there is only one variant present on the app
